[4.2.0] Add making non-admin user read-only config (#1071)
wso2/docs-mi: en/docs/install-and-setup/setup/user-stores/managing-users.md

### Non-admin users

Users that do not have admin privileges can access the management API, the CLI, and the dashboard to update the deployed artifacts, view and monitor integration artifacts and logs.

#### Restrict non-admin users from updating the deployed artifacts
To restrict the non-admin users from updating the deployed artifacts through **management API** and **CLI**, you can add the following configuration to the `<MI_HOME>/conf/deployment.toml` file. (Available from update level:4.2.0.103)

```toml
[user_access]
make_non_admin_users_read_only = true
```

> To restrict the non-admin users from updating the deployed artifacts with **MI dashboard**, you can add the same configuration to the `<MI_DASHBOARD_HOME>/conf/deployment.toml` file. (Available from update level:4.2.0.18)

Reviewer: shall we give the config here. Saying same configuration can be confusing.

Author: Updated with d67de7fab40aa6e2691877a7c4b95d656b0fab8e

### Non-admin users

> Users that do not have admin privileges can access the management API, the CLI, and the dashboard to update the deployed artifacts, view and monitor integration artifacts and logs.

Reviewer 2: ```suggestion
Users without admin privileges can access the management API, the CLI, and the dashboard to update deployed artifacts, and to view and monitor integration artifacts and logs.
```
